romancal/tweakreg/tweakreg_step.py

```diff
@@ -0,0 +1,560 @@
+"""
+Roman pipeline step for image alignment.
+
+:Authors: Mihai Cara & Mairan Teodoro
```

Comment:
Please remove `Authors`. it's a relic from an old system. We (should) maintain a contributors page.

Reply:
Done.